src/fileio.c

```diff
@@ -817,14 +817,16 @@ void writeOBANetwork(network_type *network, char *linkFileName,
  */
 void writeNetworkFlows(network_type *network, char *outputFileName) {
     FILE *outFile = openFile(outputFileName, "w");
-    int ij;
+    int ij, c;
 
     for (ij = 0; ij < network->numArcs; ij++) {
-        fprintf(outFile, "(%d,%d) %f %f\n", network->arcs[ij].tail + 1,
+        fprintf(outFile, "(%d,%d) %f ", network->arcs[ij].tail + 1,
                                             network->arcs[ij].head + 1,
-                                            network->arcs[ij].flow,
-                                            network->arcs[ij].cost
-                                                - network->arcs[ij].fixedCost);
+                                            network->arcs[ij].flow);
+        for (c = 0; c < network->numClasses; ++c) {
+            fprintf(outFile, "%f ", network->arcs[ij].classFlow[c]);
+        }
```

Comment:
Do we need this?

Reply:
This is responsible for printing the flows of each individual class, part of PD’s project. But I still think it makes sense because we print the total flow and then broken down by class.

Reply:
I agree, can we make this a debug output instead?